social.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: Media Strategy
+tagline: Jupyter Media Strategy
+permalink: /social
+---
+
+The Jupyter Media Strategy Working Group (JMS) exists to ensure that communications in Jupyter official channels are strategic and benefit Project Jupyter.  Please see our [charter](https://jupyter.org/governance/charters/MediaStrategyCharter.html) as and [current list of constituents](https://jupyter.org/governance/people.html#jupyter-media-strategy-working-group) for more information.
```

Comment:
Another small typo and wording change request on line 8. Change "as and [current list of constituents]"

to "and [current list of members"

Reply:
Fixed